symfem/polynomials.py

```diff
@@ -42,6 +42,29 @@ def polynomial_set_vector(
         for i in range(range_dim)
     ]
 
+def polynomial_set_1d_ZH(
```

Comment:
It would be better to define this in the huang zhang element file, as it's unlikely to be reused anywhere else